inference-engine/cmake/dependencies.cmake

```diff
@@ -250,6 +250,9 @@ if (ENABLE_OPENCV)
             elseif (LINUX_OS_NAME STREQUAL "Ubuntu 20.04" AND X86_64)
                 set(OPENCV_SUFFIX "ubuntu20")
                 set(OPENCV_HASH "2b7808d002864acdc5fc0b19cd30dadc31a37cc267931cad605f23f2383bfc21")
+            elseif (LINUX_OS_NAME STREQUAL "LinuxMint 20.1" AND X86_64)
```

Comment:
can we just change the previous condition to be:
```cmake
elseif ((LINUX_OS_NAME STREQUAL "Ubuntu 20.04" OR LINUX_OS_NAME STREQUAL "LinuxMint 20.1") AND X86_64)
                set(OPENCV_SUFFIX "ubuntu20")
                set(OPENCV_HASH "2b7808d002864acdc5fc0b19cd30dadc31a37cc267931cad605f23f2383bfc21")
elseif(...)
```

Reply:
Later, you can set different hash for Ubuntu and Mint, or not?

Reply:
If these OSes are compatible now, no needs to provide different packages in future I suppose

Reply:
ok)

Reply:
Could you please update a patch?